spec/factories/report.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :v2_report, class: 'V2::Report' do
+    to_create do |instance, context|
+      instance.attributes = V2::Policy.find(context.policy_id).attributes
+      instance.reload
+    end
+
+    transient do
+      policy_id { nil }
```

Comment:
I feel like this needs a bit more thinking, basically this one never creates a report, just enhances a policy. The factory should be self-contained, creating a policy+tailoring+system+test_results all in one and all its dependants could/should be parametrized. Basically most of the logic you have in the `let(:items)` above should live here somehow.